meson.build

```diff
@@ -393,10 +393,10 @@ endif
 # ml-agent
 ml_agent_dep = dependency('', required: false)
 if not get_option('ml-agent-support').disabled()
-  ml_agent_dep = dependency('ml-agent', required: false)
+  ml_agent_dep = dependency('mlops-agent', required: false)
```

Comment:
how about changing 'features' to check ml-agent library?
```
  'ml-agent-support': {
    'target': 'mlops-agent',
    'project_args': { 'ENABLE_ML_AGENT' : 1 }
  },

dependency to 'ml_agent_support_deps'
```

ml-agent has pkgconf, so I think finding library or header file is unnecessary.

Reply:
Thanks for the valuable comment. I applied it.